drivers/SmartThings/matter-sensor/fingerprints.yml

```diff
@@ -33,6 +33,17 @@ matterManufacturer:
     vendorId: 0x1021
     productId: 0x0001
     deviceProfileName: motion-contact-battery
+  #Heiman
+  - id: "Heiman Motion"
+    deviceLabel: Heiman Motion
+    vendorId: 0x120B
+    productId: 0x1001
+    deviceProfileName: matter-motion-battery-illuminance
+  - id: "Heiman DoorAndWindow"
```

Comment:
In accordance with the rest of the fingerprints, should this id be `id: "Heiman/DoorAndWindow"` (see [id: "Eve/DoorAndWindow](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1026/files#diff-d421b34194f8b655303df53e5d9d5d991a51cef4fb3882b9f9858aae830074b7L14)", [id: "TUO/ContactDoorAndWindow"](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1026/files#diff-d421b34194f8b655303df53e5d9d5d991a51cef4fb3882b9f9858aae830074b7L25))

Reply:
we don't really have hard and fast rules about what the id for fingerprints should be, and they're not user-facing